man/mono.1

```diff
@@ -321,6 +324,13 @@ app.exe.config available when compiling lib.dll ahead of time, use:
 	mono --apply-bindings=app.exe.config --aot lib.dll
 .fi
 .TP
+\fB--assembly-loader=MODE\fR
+If mode is \fBstrict\fR, Mono will check that the public key token, culture and version
+of a candidate assembly matches the requested strong name.  If mode is \fBlegacy\fR, as
+long as the name matches, the candidate will be allowed. \fBstrict\fR is the behavior
+specified by the standard but may break some existing mono-based applications.
```

Comment:
"standard" is the wrong word, I think.  What we're really matching is the [behavior of .NET Framework](https://msdn.microsoft.com/en-us/library/yx7xezcf(v=vs.110).aspx)